[Ref Guide] - Theme variants changes (#731)
dojo/framework: docs/en/styling/introduction.md

});
```

## Using a theme variant within a widget

> -   Set the `theme.variant` class on your widget `root`.

Reviewer: ```suggestion
-   Set the `theme.variant` class on the widget's `root`.
```

> -   css-properties get applied at the correct dom level and do not leak out of your widget dom.

Reviewer: ```suggestion
-   css-properties get applied at the correct DOM level and do not leak out of the widget's DOM.
```

**Note:** When using both function-based and class-based widgets, the theme needs to be registered with the application registry. This is true when using any class-based widget dependencies such as `@dojo/widgets`. Please see the [class-based theming section]() for more details.

## Setting the theme variant

> If using a theme with `variants`, the `default` variant will be selected automatically. If you wish to set a different variant, you can do that via the `theme.set` function. The variant name passed must be a key of the themes exported `variants`.

Reviewer: ```suggestion
If using a theme with `variants`, the `default` variant will be selected automatically. Use the `theme.set` function to set a different variant - the variant name passed must be a key of the theme's exported `variants`.
```
